gnomad_constraint/plots/generate_constraint_plots.R

```diff
@@ -243,3 +309,226 @@ for (version in names(datasets)) {
     ggsave(p, filename = plot_path, dpi = 300, width = 8, height = 6, units = "in")
   }
 }
+
+
+####################################################################
+####################################################################
+# Plot LOEUF decile change from v2 to v4
+####################################################################
+####################################################################
+# Filter to MANE Select Ensembl transcripts that are present in both v2 and v4
+# and that have no constriant flags
+decile_data <- filter(
+  constraint_data,
+  (.data$v2 == TRUE &
+    .data$v4 == TRUE &
+    .data$constraint_flags == "[]" &
+    .data$mane_select == "true" &
+    grepl("^ENST", transcript))
+)
+decile_plot <- plot_decile_change(decile_data)
+
+plot_path <- get_plot_path(
+  "decile_change",
+  output_basedir = output_basedir
+)
+ggsave(
+  decile_plot,
+  filename = plot_path,
+  dpi = 300,
+  width = 8,
+  height = 4,
+  units = "in"
+)
+
+
+####################################################################
+####################################################################
+# Plot LOEUF and Z-Score comparison between v2 and v4
+####################################################################
+####################################################################
+# Filter to MANE Select Ensembl transcripts that are present in both v2 and v4
+# and that have no constriant flags
+comparision_df <- filter(
+  constraint_data,
+  (.data$v2 == TRUE &
+    .data$v4 == TRUE &
+    .data$constraint_flags == "[]" &
```

Comment:
If using other suggestion
```suggestion
    .data$constraint_flags.v4 == "[]" &
```